fancycompleter.py

```diff
@@ -128,6 +128,11 @@ class DefaultConfig:
         complex: Color.yellow,
         bool: Color.yellow,
     }
+    # Fallback to look up colors by `isinstance` when not matched
+    # via color_by_type.
+    color_for_isinstance = [
```

Comment:
@antocuni 
Any feedback on the config name?
Would `color_for_instances` be better?

Reply:
what about `color_by_baseclass`?

Reply:
Ok.